### PR TITLE
1 Update sommaire on readme

### PR DESCRIPTION
**Pull request by Fr-Dae**
https://github.com/Fr-Dae/antimicrox/issues/1
https://github.com/AntiMicroX/antimicrox/issues/1087
glossaire for readme and fix 3 broken link (anchor)

### CODE REVIEW

**Comment by pktiuk** [comment]
I don't find these changes in the readme useful. They don't improve readability and just make content list too long.

Just look at the beginning of your readme:

obraz

There is already one line under the name of the app. Adding another one just  wastes space.

**Comment by Fr-Dae** [comment]
you know, you can asking for review change, it's more visible than a a comment on a pr or a issue 

it's fix

**Comment by Fr-Dae** [comment]
the goal of one glossair it's to navigate quicky, because dokuwiki and mermaid generate it automatcly, but github don't (sadly) so we need make it mnually, i have zero skill in C++, but i can contribute on readme, fix link and tasks like that, like i have contribute to antimicro long time ago

File: README.md [hunk 1]
```diff
@@ -1,14 +1,23 @@
...
-4. [Wiki](#wiki)
-5. [Command Line](#command-line) 
-6. [Testing Under Linux](#testing-under-linux)  
-7. [AntiMicroX Profiles](#antimicrox-profiles)
-8. [Support](#support)  
-
+- [Description](#description)
+- [License](#license)
+- [Installation](#installation)
+	-  [windows](#windows)
```
Comment (pktiuk) [inline README.md:5]: You just added a glossary entry to paragraph, which takes literally 2 lines.

This does not make sense. That's why installation section does not have links to its subsections

**Comment by pktiuk** [comment]
> the goal of one glossair it's to navigate quicky, because dokuwiki and mermaid generate it automatcly, but github don't (sadly) so we need make it mnually, i have zero skill in C++, but i can contribute on readme, fix link and tasks like that, like i have contribute to antimicro long time ago

If you don't know C++ you can still contribute in the places where contributions make sense. For example you can update [Wiki](https://github.com/AntiMicroX/antimicrox/wiki) or even better update AntiMicroX's web page: https://github.com/AntiMicroX/antimicrox.github.io (this one may be a bit bigger task, because in current state this website is not very useful)

**Comment by pktiuk** [comment]
Of course if you know how to make usable websites I would highly recommend taking a look at the website.

**Comment by Fr-Dae** [comment]
> Of course if you know how to make usable websites I would highly recommend taking a look at the website.

the website need more illustration and image, but it's good

**Comment by pktiuk** [comment]
> > Of course if you know how to make usable websites I would highly recommend taking a look at the website.
> 
> the website need more illustration and image, but it's good

Not really it needs a lot of things like easy Download button for users or description of features. You can check issues in that repo for context: https://github.com/AntiMicroX/antimicrox.github.io/issues/1 https://github.com/AntiMicroX/antimicrox.github.io/issues/4

**Pull request closed without merging**